sdk/src/wallet/core/operations/stronghold_backup/mod.rs

```diff
@@ -103,16 +107,24 @@ impl Wallet {
         let ignore_backup_values = ignore_if_bip_path_mismatch.map_or(false, |ignore| {
             if ignore {
                 // TODO: #1279 okay that if both are none we always load the backup values?
-                wallet_data.bip_path != read_bip_path
+                curr_bip_path != read_bip_path
             } else {
                 false
             }
         });
 
+        // TODO: Do we even need to do this, since it should get set when we restore below?
```

Comment:
I think it's a bit different, it only gets set below `if restore_wallet {` which depends on the bech32 hrp